dipy/denoise/patch2self.py

```diff
@@ -145,16 +145,18 @@ def vol_denoise(
     ----------
     data_dict : dict
         Dictionary containing the following
-        data_name : str
+
+        - data_name : str
```

Comment:
@iprajwalreddy This was not checked after merging and prior to committing. Same applies to other parameters in this docstring.

Reply:
![image](https://github.com/user-attachments/assets/d894745f-3f2b-4aa3-8614-bdbc3efc9245)

I am committing a new code which fixes it like this. Please let me know if you think it requires any changes.

Reply:
Check the new commit, as Sphinx may consider the indented block as a note.